### PR TITLE
Make pytz optional

### PR DESCRIPTION
**Pull request by 9128305**
## Description of the Change
Pytz should be an optional dependency.
## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [ ] documentation updated
- [ ] `CHANGELOG.md` updated (only for user relevant changes)
- [ ] author name in `AUTHORS`

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -60,6 +60,7 @@ deps =
     pytest-xdist
     pytest-mock
     requests
+    pytz
```
Comment (9128305) [inline tox.ini:63]:
```suggestion
    pytz; python_version < '3.9'
```

**Comment by n2ygk** [comment]
You will need to rebase this as master has recent PRs merged in.

**Comment by 9128305** [comment]
Are you squashing merge requests? I could update without manually rebasing

**Comment by n2ygk** [comment]
> Are you squashing merge requests? I could update without manually rebasing

Yes.

**Comment by 9128305** [comment]
Done.

**Review by n2ygk — APPROVED** [review]
thanks

**Pull request merged**